pkg/config/store/convert.go

```diff
@@ -38,5 +39,9 @@ func convert(spec map[string]interface{}, target proto.Message) error {
 	if err != nil {
 		return err
 	}
-	return jsonpb.Unmarshal(bytes.NewReader(jsonData), target)
+	if err = jsonpb.Unmarshal(bytes.NewReader(jsonData), target); err != nil {
+		glog.Warningf("unable to unmarshal: %s, %s", err.Error(), string(jsonData))
```

Comment:
I'm not sure if this was added for your debugging, or this should be in production. Normally, simply returning an error would be sufficient (assuming that `err` contains information about name of property or types), unnecessary to dump the `jsonData`.

If you want to ensure `jsonData` in the error, maybe this line should return a new error wrapping `err`, as `fmt.Errorf("unable to unmarshal: %s, %s", err.Error(), string(jsonData))` rather than logging directly.